contributing.md

```diff
@@ -2,8 +2,34 @@
 
 ## Build-related commands
+
+If you are using a shell capable of running Bash scripts, run from the project root directory:
+
+| &#x23; | Command                                             | Description                                                                                                                                                                                                                                                                                                                                   |
+|--------|-----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0      | `source ./activate-venv.sh`                         | Creates or in the case existence activates Python virtual environment for the project &#8212; `ttt-venv`. Also adds the project root directory to `PYTHONPATH` environment variable. `toolchain-requirements.txt` contains all Python dependencies of the project. `requirements.txt` contains all dependencies required for the application. |
+| 1      | `./pylint-check.sh && ./mypy-check.sh && ./test.sh` | Analyzes and reports errors, checks style, checks static types, and runs tests.                                                                                                                                                                                                                                                               |
+| 1.1    | `./pylint-check.sh`                                 | Analyzes and reports errors, checks style.                                                                                                                                                                                                                                                                                                    |
+| 1.2    | `./mypy-check.sh`                                   | Checks static types.                                                                                                                                                                                                                                                                                                                          |
+| 1.3    | `./test.sh`                                         | Runs all tests.                                                                                                                                                                                                                                                                                                                               |
```

Comment:
These 3 commands are hardly useful as anyone can derive them trivially from the previous command. Let's remove them to avoid overloading the table.

Reply:
Done.